raiden/network/transport/udp/udp_transport.py

```diff
@@ -207,7 +208,7 @@ def __init__(self, discovery, udpsocket, throttle_policy, config):
     def start(
             self,
             raiden: RaidenService,
-            queueids_to_queues: typing.Dict[QueueIdentifier, typing.List[Event]],
+            queueids_to_queues: typing.Dict[QueueIdentifier, typing.List[Message]],
```

Comment:
You should use [QueueIdsToQueues](https://github.com/raiden-network/raiden/blob/4d4f6d5c593c418bd7c4ff0f3784415569c5af43/raiden/transfer/state.py#L22) here, and change the type there